scripts/verify.sh

```diff
@@ -37,18 +31,14 @@ golangci-lint run --disable-all \
     --enable=ineffassign \
     --enable=nakedret \
     --enable=interfacer \
-    --enable=misspell \
     --enable=dupl \
     --enable=goimports \
     --enable=gocyclo \
 
 
 ##todo(camilamacedo86): The following checks requires fixes in the code
 # --enable=golint
-# --enable=gocyclo
 # --enable=lll
 # --enable=goconst
 # --enable=gosec
 # --enable=maligned
-
-GO111MODULES=on go list -mod=readonly ./...
```

Comment:
Any reason for removing this?

Reply:
has no need .

Reply:
It is just lisring the modules in Travis, not needed, just consumes resources (I know, its negligible but...)